Makefile.in

```diff
@@ -439,7 +434,10 @@ release: version
 	$(MAKE) clean
 
 .PHONY: dist
-dist: release
+dist:
```

Comment:
Perhaps rename this target to `binary-release`?  And, you shouldn't name the resulting `.tgz` a `.src.tgz`.

Reply:
Thanks for your comments. I was thinking of the standard `dist` target that we should expose one way or another, but for now adopted your suggestion.